tests/fixtures/rules/prefer-class-directive/invalid/_config.json

```diff
@@ -0,0 +1,3 @@
+{
+	"options": [{ "prefer": "always" }]
+}
```

Comment:
Could you please add test cases to check what is reported as an error when using `empty`?

Reply:
I've added some specific tests, expanding on test 01.
But note that as `prefer: "empty"` is a relaxation of the rule, the other tests cover it quite well.
While doing that I noticed that the 'empty' rule should also apply when the classname contains just whitespace - so I rejigged my changes to handle that too.